content/joysticks/winwing/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: WINWING
+weight: 50
+---
+
+MobiFlight supports the WINWING FCU, MCDU, and PFP3N.
+
+> [!TIP]
+> WINWING devices are automatically shown in the input and output configuration dialogs. They will not appear in the **Modules** tab of the **Settings** dialog as they are not [boards](/boards/), to confirm the WINWING device is seen by MobiFlight, use the **Peripherals** tab of the **Settings** dialog.
+
```

Comment:
Since this is the main landing page for WINWING devices we should take advantage of it to consolidate key information for new users.

The Supported Aircraft for the CDU section can be moved from a standalone page and added here in a specific section for the CDU so people don't have to navigate to a separate page to find it.

Reply:
I expect more special information like for PMDG. FlyByWire and CJ4 will have extra explanations. Probably CRJ as well, and there also will be a section for the ProSim planes. I did not want the landing page to grow to long.

Maybe put the supported aircraft table on the main landing page and then have another linked page with "detailed aircraft information" or something like that?

Reply:
Yes, that would work great and would be consistent with how we did MIDI devices. See https://docs.mobiflight.com/midi-devices/ for the landing page, and https://docs.mobiflight.com/midi-devices/configuring-device/ for how we handled configuring different devices.